event-message/event-message.bal

```diff
@@ -24,9 +24,9 @@ service /api/v1 on new http:Listener(8080) {
         string body = string `Incident ${req.incident.description} reported: 
                               ${req.incident.date} at ${req.incident.time}.`;
         http:Request twilioReq = new;
-        string payload = string `From=${check url:encode(FROM_NO, "utf-8")}
-                                 &To=${check url:encode(req.phoneNo, "utf-8")}
-                                 &Body=${check url:encode(body, "utf-8")}}`;
+        string payload = string `From=${check url:encode(FROM_NO, "utf-8")}` +
+                         string `&To=${check url:encode(req.phoneNo, "utf-8")}` +
+                         string `&Body=${check url:encode(body, "utf-8")}}`;
```

Comment:
Extra `}` in line 29? please make sure it runs.

Also feels like string template is not a helping anymore. let's uses `+`
```suggestion
        string payload = "From=" + check url:encode(FROM_NO, "utf-8") +
                         "&To=" + check url:encode(req.phoneNo, "utf-8") +
                         "&Body=" + check url:encode(body, "utf-8");
```